source/docs/v3/tenancy-bootstrappers.blade.md

```diff
@@ -29,31 +29,76 @@ php artisan cache:clear --tag=tenant_123
 Note that you must use a cache store that supports tagging, e.g. Redis.
 
 ## Filesystem tenancy bootstrapper {#filesystem-tenancy-boostrapper}
+The filesystem bootstrapper makes your app's `storage_path()` and `asset()` helper and the `Storage` facade tenant-aware by modifying the paths they retrieve.
 
-This bootstrapper does the following things:
+> Note: If you want to bootstrap filesystem tenancy differently (e.g. provision an S3 bucket for each tenant), you can absolutely do that. Take a look at the package's bootstrappers to get an idea of how to write one yourself, and feel free to implement it any way you want.
+
+### Storage path helper
+
+The bootstrapper suffixes the path retrieved by `storage_path()` to make the helper tenant-aware.
+
+- The suffix is `config('tenancy.filesystem.suffix_base') . $tenantKey`
+- The suffix base is 'tenant' by default, but feel free to change it in the tenancy config
+- After suffixing the `storage_path()` helper, it returns `"/$path_to_your_application/storage/$suffix/"`
```

Comment:
This one seems a bit confusing, since `$suffix` here is the suffix base + tenant id, but the readers may not get that without seeing an example. So perhaps we could replace this by an example with a random tenant id and the full generated path?

Reply:
That might, be better, I'll update it and we'll see

Reply:
Added the example, but I'd keep the `config('tenancy....'). $tenantKey` because that can make things a bit clearer.

Reply:
Looks like this isn't fully resolved. I meant that `/$path_to_your_application/storage/$suffix/` should be `/$path_to_your_application/storage/tenant42/` given the example values you added. The examples are great but I'd make this change as well to make the final string in the example complete, rather than having variables in it (with the exception of `$path_to_your_application` which we of course don't know).

Reply:
Created a suggestion that you can just commit and then you can resolve this https://github.com/stancl/tenancy-docs/pull/201/files#r979141050